packages/react-environment/src/WebbProvider.tsx

```diff
@@ -294,13 +305,14 @@ export const WebbProvider: FC<WebbProviderProps> = ({ applicationName = 'Webb Da
         let defaultAccount = networkDefaultConfig[chainConfig.id]?.defaultAccount;
         defaultAccount = defaultAccount ?? accounts[0]?.address;
         const defaultFromSettings = accounts.find((account) => account.address === defaultAccount);
+        console.log(defaultFromSettings, 'defaultFromSettings');
         if (defaultFromSettings) {
           _setActiveAccount(defaultFromSettings);
           await activeApi.accounts.setActiveAccount(defaultFromSettings);
           networkStorage?.set('networksConfig', {
             ...networkDefaultConfig,
             [chainConfig.id]: {
-              ...chainConfig[chainConfig.id],
```

Comment:
Why this change here?

Reply:
This if for change to be just `...chianConfig` , on init restoring the default account or the first one on the accounts list to the storage